src/fibonacciBases.js

```diff
@@ -0,0 +1,47 @@
+function fibonacci(n) {
+  if (n === 0 || n === 1) {
+    return 1;
+  } else {
+    return fibonacci(n - 2) + fibonacci(n - 1);
+  }
+}
+
+function decToFibonnaci(number) {
```

Comment:
what if you make `binaryToNumber`, `numberToBinary`, `FinbonnaciBinaryToNumber` and `numberToFinbonnaciBinary`?